docs/resources/permissions.md

```diff
@@ -315,6 +315,118 @@ resource "databricks_permissions" "token_usage" {
 }
 ```
 
+## SQL Endpoint Usage
+
+[SQL endpoints](https://docs.databricks.com/sql/user/security/access-control/sql-endpoint-acl.html) have two possible permissions:  `CAN_USE` and `CAN_MANAGE`:
+
+```hcl
+resource "databricks_group" "auto" {
+    display_name = "Automation"
+}
+
+resource "databricks_group" "eng" {
+    display_name = "Engineering"
+}
+
+resource "databricks_permissions" "endpoint_usage" {
+    sql_endpoint_id = "3244325"
```

Comment:
Should we use the resource IDs instead of strings?

Reply:
it's WIP 😄